benchmarking/smartseq2/merge_expression_matrix/merge_table_quants.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import pandas as pd
+import numpy as np
+from google.cloud import storage
+import json
+from os.path import basename
+import sys
+import requests
+
+
+def merge_expression_matrix(uuid, run_name,output_name,value_name):
+    """
+    pipeline produce gene quantification by using either FeatureCounts or RSEM at single sample level 
+    This function is called to merge or aggregate all the single cell/sample level results into
+    a single matrix file
+    param uuid: the uuid of pipeline run, top level uuid 
+    param run_name: run_name is the output name in pipeline metadata, for exampel, run_pipelines.RunHisat2Pipeline.unq_genes_counts
+    param output_name: output file name, ex STAR_tpm.csv
+    para value_name: aggregate value type, can be est_count or tpm. 
+    """
+    # set up auth
+    client = storage.Client()
+    bucket = client.get_bucket('broad-dsde-mint-dev-cromwell-execution')
+    # load cromwell credential
+    logins = json.load(open('/usr/secrets/broad-dsde-mint-dev-cromwell.json'))
+    # meta_url
+    metadata_url = "https://cromwell.mint-dev.broadinstitute.org/api/workflows/v1/"+uuid+"/metadata?expandSubWorkflows=false"
```

Comment:
Is it necessary to parameterize above parameters? It seems all of them are hard-coded now.